supplier_app/views.py

```diff
@@ -135,19 +135,26 @@ def get_context_data(self, **kwargs):
 class CreateTaxPayerView(SupplierWithoutCompanyMixin, TemplateView, FormView):
     template_name = 'supplier_app/taxpayer-creation.html'
 
-    def get(self, request, *args, **kwargs):
-        context = {
+    def get_context_data(self, **kwargs):
+        """
+        Insert the form into the context dict.
+        """
+        kwargs.update({
             'address_form': AddressCreateForm(),
-            'taxpayer_form': TaxPayerCreateForm(self.request.user),
+            'taxpayer_form': TaxPayerCreateForm(),
             'bankaccount_form': BankAccountCreateForm(),
-        }
-        return self.render_to_response(context)
+        })
+        return kwargs
 
     def post(self, request, *args, **kwargs):
         forms = {
-            'taxpayer_form': TaxPayerCreateForm(request.user, self.request.POST),
-            'address_form': AddressCreateForm(self.request.POST),
-            'bankaccount_form': BankAccountCreateForm(self.request.POST),
+            'taxpayer_form': TaxPayerCreateForm(
+                data=request.POST,
+                files=request.FILES),
+            'address_form': AddressCreateForm(data=request.POST),
+            'bankaccount_form': BankAccountCreateForm(
```

Comment:
this variable could be renamed to: `bank_account_form`?

Reply:
Yes you are right